File: poo/Cpp/AprovadoReprovado/AprovadoReprovado.cpp
#include <iostream>

int main() {
    double m;
    int f;
    std::cin >> m >> f;
    if (m >= 6.0 && f <= 30) {
        std::cout << "Aprovado!" << std::endl;
    } else {
        std::cout << "Reprovado!" << std::endl;
    }
    return 0;
}

File: poo/Java/AprovadoReprovado/AprovadoReprovado.java
import java.util.Scanner;

public class AprovadoReprovado {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double m = scanner.nextDouble();
        int f = scanner.nextInt();
        if (m >= 6.0 && f <= 30) {
            System.out.println("Aprovado!");
        } else {
            System.out.println("Reprovado!");
        }
        scanner.close();
    }
}